sky/provision/gcp/instance.py

```diff
@@ -1,8 +1,11 @@
 """GCP instance provisioning."""
 import collections
+import re
 import time
 from typing import Any, Callable, Dict, Iterable, List, Optional, Type
 
+from googleapiclient import errors
```

Comment:
We should not directly import the googleapiclient, to avoid people who do not have GCP set fails due to import error.

Instead we should use
```
from sky.adaptors import gcp

except gcp.http_error_exception() as e:
```